MAFIA/Views/Controllers/Game/GameViewController.swift

```diff
@@ -27,6 +27,7 @@ class GameViewController: UIViewController {
             updateGameUI()
         }
     }
+    private var pullToRefresh: UIRefreshControl!
```

Comment:
It's better to put the global variables together in the top of the view controller with the rest of the global variables and outlets.

Reply:
It was in the global variables, well done 😂

Reply:
It was in the global variables, well done 😂